tests/dat/actions/echo-web-http.js

```diff
@@ -2,11 +2,9 @@
 // license agreements; and to You under the Apache License, Version 2.0.
 
 function main(params) {
-    var bodyobj = params || {};
-    bodystr = JSON.stringify(bodyobj);
     return {
         statusCode: 200,
         headers: { 'Content-Type': 'application/json' },
-        body: new Buffer(bodystr).toString('base64')
+        body: params
```

Comment:
should this be `JSON.stringify(params)` instead?

Reply:
`body` can be `string` or `object`, today we support `object` and want to make sure we continue to support it.
With this PR what we are doing is in addition of supporting a `dict`/`object`  to also allow `string`.

or I'm missing something else?

Reply:
im just trying to recover the original behavior where body was returning a string instead of a JSON object.